packages/ml_api/docker_test/.env.sample

```diff
@@ -0,0 +1,5 @@
+DB_USER=gvenik_test
+DB_PASSWORD=password
+DB_PORT=7617
+DB_HOST=0.0.0.0
+DB_NAME=just_a_db
```

Comment:
set `DB_USER = test_user`, `DB_USER = test_db`. Also, maybe we put a different placeholder for `DB_PORT` than 7617, say 7618? It is not a problem but I guess it is better to have different port placeholders for docker and docker test.